ansible/playbooks/roles/maps/files/c3nav-gunicorn.service

```diff
@@ -6,7 +6,7 @@ After=network.target
 User=c3nav
 Group=c3nav
 WorkingDirectory=/opt/c3nav/src
-ExecStart=/opt/c3nav/env/bin/gunicorn --workers 8 --bind unix:/var/run/c3nav/gunicorn-c3nav.sock c3nav.wsgi:application --log-level warning --error-logfile /var/log/c3nav/gunicorn.log --access-logfile /dev/null
+ExecStart=/opt/c3nav/env/bin/gunicorn --workers 4 --bind unix:/var/run/c3nav/gunicorn-c3nav.sock c3nav.wsgi:application --log-level warning --error-logfile /var/log/c3nav/gunicorn.log --access-logfile /dev/null
```

Comment:
It might be good to do something like `{{ ansible_processor_vcpus }}` to make the workers match the number of CPUs on the system. Even with threads, you're likely to be limited to the GIL as a worker CPU limit.